lib/pacemaker/pcmk_sched_resource.c

```diff
@@ -755,7 +755,7 @@ apply_parent_colocations(const pe_resource_t *rsc, GHashTable **nodes)
         other->cmds->add_colocated_node_scores(other, rsc->id, nodes,
```

Comment:
The for-loop before this should use pcmk__coloc_select_any instead of 0. I can't comment on that one directly since it's not in the commit, but this is a good time to fix it.

Reply:
Good point